### DISCUSSION TITLE
Licensing and perspective of open use

### DISCUSSION THREAD

**Discussion by Shoadi** [anyproto/contributors#81] [Self-hosting]
Hello fellow entities,

I am evaluating the use of Anytype for work with a self-hosted backup-server, and therefore have some questions about the licensing model as well as its future perspectives.

According to: 
https://networks.any.coop/
`"Non-Commercial Use" means personal, academic, scientific, or research and development use, or evaluating the Software, but does not include uses where the Software facilitates any transaction of economic value other than on Allowed Networks.`
...I would believe, that me and my working colleagues could write and store our documentations and concepts (...which of course are of economic value...) locally (no transaction) - but at the moment we would upload (transact) to our self-hosted backup-server - it would count as kind of a commercial use - is that right?

Further, based on the licensing-model, I would suppose, that in the future there will be different networks for the exchange of data. Can someone maybe evaluate a bit on what is planned here? 

If we implement Anytype right now into our workflow (and therefore store our Knowledge there) -will we in the future be able to collaborate on that, share and sync our stuff, or will that be only possible on licensed networks?

Thanks a lot in advance and have a great day.

**Comment by fuksman** [comment]
Hello @Shoadi, 

Please see the [FAQ](https://legal.any.coop/?id=can-i-self-host-anytype-within-my-company), I believe it covers your case: if you self-host without requiring subscriptions/payments/staking/etc., then it complies with the license terms.

Even now, different networks exist: our main production network and all self-hosted networks. As we aim to protect the published source code from rivals' abuse, we crafted this license to ensure that nobody can charge users for the product just by copying its source code. 

We see the evolution of networks in different directions, for example, we can introduce the role of service providers who would be able to host networks with allowed commercial transactions. We're also interested in providing some federative features for any-sync networks. Let's say you can self-host your company data but still access some other spaces from other networks. It's important to note here that this is not a part of our current roadmap, just ideas for future development.

I hope this answers your questions. 

We're deeply interested in business users of Anytype. We would appreciate it if you could answer some of our questions so that we can better understand your product needs. If you're okay with having a short call or email conversation, please contact me via fuksman@anytype.io or LinkedIn.
